src/components/ui/Pool/PoolDeposit.tsx

```
@@ -429,332 +441,332 @@ const PoolDeposit = ({
-                  <Typography secondary tt="uppercase">
-                    {BigInt(
-                      ethers.parseUnits(
-                        String(inputValueToken1.toFixed(selectedPool.token1.decimals) || 0),
-                        selectedPool.token1.decimals
-                      )
-                    ) > balanceToken1
-                      ? "Not Enough Balance"
-                      : "Deposit"}
-                  </Typography>
-                </Button>
+                        : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)